EveripediaNetwork/ep-ui
# fix profile on search in accounts page

# Fix the search dropdown showing under the sticky profile on the account page

_Search dropdown when the account page was showing under the sticky profile which is not a good UI practice and has been resolved_

## How should this be tested?

1. Go to any account profile, either click on the search bar or scroll down on the page
2. When the sticky profile appears it should be under the search dropdown

<details>
  <summary>Before</summary>
  
<img width="1920" alt="Screenshot 2024-08-30 at 11 36 37 AM" src="https://github.com/user-attachments/assets/96238c3c-101b-4bb3-8678-b26e27549d56">

</details>

<details>
  <summary>After</summary>
  
<img width="1920" alt="Screenshot 2024-08-30 at 11 36 57 AM" src="https://github.com/user-attachments/assets/24164103-b95f-4f48-99f0-67aa943a6744">
</details>

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3097
